tests/lib/jsonrpc_client_test_base.py

```diff
@@ -39,6 +39,9 @@ class JsonRpcClientTestBase(unittest.TestCase):
         b'{"id": 0, "result": 123, "error": null, "status": 1, "uid": 1, '
         b'"callback": "1-0"}')
     MOCK_RESP_WITH_ERROR = b'{"id": 0, "error": 1, "status": 1, "uid": 1}'
+    MOCK_REST_LONG_STRINGS = bytes((
```

Comment:
`bytes('{"id": 0, "result": "' + 'Very long long long long response. ' * 40 + '", "error": null, "status": 0, "callback": null}', 'utf-8')`
we prefer to use this format, but it doesn't work in py2.
add a TODO for py2 deprecation later.